facebook/Makefile

```diff
@@ -5,9 +5,11 @@ DRY:=yes
 ifeq ($(DRY),yes)
 	OUTSIDE_EMAIL_FLAGS:=-d
 	SFTP_POST:=echo -e "Would run: sftp -b <(echo -e \"\$${BATCH}\") -P 2222 fb-automation@ftp.delphi.cmu.edu\n$${BATCH}"
+	CONTINGENCY_SFTP_POST:=echo -e "Would run: sftp -b <(echo -e \"\$${BATCH}\") ???@publishing.andrew.cmu.edu\n$${BATCH}"
```

Comment:
Will need to set up a daemon account for the AWPS website, preferably with exclusive auto-publishing capabilities.

Reply:
I have an open ticket with facilities on this; they don't currently support it but they're discussing it today and will let us know. we may need to fall back to a delphi-operated hosting solution.

Reply:
Update: They've started working on a tentative solution. We will however need to create a dedicated AWPS collection to host them, since there's no way to set up autopublish on just part of a collection.

Reply:
CCing @capnrefsmmat

Reply:
delphi team leads just approved turning on autopublish for delphi-web and just making sure everyone with push access knows